common/djangoapps/third_party_auth/models.py

```diff
@@ -347,6 +347,9 @@ class SAMLProviderConfig(ProviderConfig):
     attr_email = models.CharField(
         max_length=128, blank=True, verbose_name="Email Attribute",
         help_text="URN of SAML attribute containing the user's email address[es]. Leave blank for default.")
+    include_in_saml_pull = models.BooleanField(
```

Comment:
I wonder if we should name this something else? @fredsmith raised a very good point that this could be abused to turn off pulling metadata when an error occurs. While I think our intentions are pure, maybe we should name this something more close to the actual use case; "manually configure SAML metadata", perhaps?